src/ploneorg/addonlisting/browser/static/w3.css

```diff
@@ -0,0 +1,210 @@
+﻿/* W3.CSS 4.04 Apr 2017 by Jan Egil and Borge Refsnes */
```

Comment:
why a separate CSS file?

especially as you use a reset / normalize part in it, which could break the Plone theme.

also it is not good to just copy a complete file from another project, there might be licence issues.